### PR TITLE
make sure colorbar points are lists of first point and update test

### PR DESCRIPTION
**Pull request by Kully**

### CODE REVIEW

**Comment by Kully** [comment]
@theengineear

**Comment by Kully** [comment]
@theengineear So Emilia pointed out that the colorbar I added to the trisurfs doesn't appear when online is enabled. I have fixed this by switching the `x[0]` to a `x[0:1]` so as to make sure `x` receives a list

File: plotly/tools.py [hunk 1]
```diff
@@ -3352,9 +3352,9 @@ def _trisurf(x, y, z, simplices, show_colorbar, edges_color,
             colorscale = FigureFactory._convert_colorscale_to_rgb(colorscale)
 
             colorbar = graph_objs.Scatter3d(
-                x=x[0],
-                y=y[0],
-                z=z[0],
+                x=x[0:1],
+                y=y[0:1],
+                z=z[0:1],
```
Comment (theengineear) [inline plotly/tools.py]: 🐄 `z[:1]` works too, fwiw.

**Comment by theengineear** [comment]
💃

**Comment by Kully** [comment]
> 💃

Excellent! Shall I update v. num?

**Comment by theengineear** [comment]
Yah, good call :)

**Comment by Kully** [comment]
> Yah, good call :)

Okay, will get to it tomorrow most lik

**Comment by Kully** [comment]
@theengineear Do you mind looking at this failing test when you have a moment? It looks like a proper image is not being produced but the only error after `nosetests` on my end is the one you said was because my matplotlib version is unsupported:

```
FAIL: plotly.tests.test_optional.test_matplotlylib.test_subplots.test_blank_subplots
----------------------------------------------------------------------
Traceback (most recent call last):
  File "//anaconda/lib/python2.7/site-packages/nose/case.py", line 197, in runTest
    self.test(*self.arg)
  File "/Users/adamkulidjian/Desktop/Adam/Plotly/plotly.py/plotly/tests/test_optional/test_matplotlylib/test_subplots.py", line 37, in test_blank_subplots
    assert equivalent, msg
AssertionError: ['yaxis1']['range'] = [0.80000000000000004, 2.0] should be [1.0, 2.2]
```

It failed this way for Python 2 and 3

**Comment by Kully** [comment]
> Do you mind looking at this failing test when you have a moment?

Nevermind, tests passed!

**Pull request merged**